### DISCUSSION TITLE
表单中，能否在任意一个输入框，回车触发表单的提交

### DISCUSSION THREAD

**Discussion by seepine** [tusen-ai/naive-ui#6142] [Q&A]
是否有参数配置一下就行，不需要自己每个input去监听keyup事件

**Comment by Sepush** [comment]
没有 而且这样的做法是不合理的

**Reply by seepine** [reply]
不合理吗，感觉很常用啊，比如登录的表单，或者表格的表单，遇到好多系统都是回车就直接提交了，类似button type=submit
